python/ray/data/_internal/execution/operators/map_transformer.py

```diff
@@ -42,6 +70,61 @@ class MapTransformFnCategory(Enum):
     PostProcess = 2
 
 
+class _MapActorContext:
```

Comment:
nit, we can remove this context wrapper and directly put these attributes in `CallableClassUDFMapTransformFn` and `AsyncCallableClassUDFMapTransformFn`.  
Because some attributes are only relevant to the async fn.